Cabal/src/Distribution/Simple/SetupHooks/Rule.hs

```diff
@@ -612,9 +610,9 @@ mkCommand
   -> StaticPtr (arg -> res)
   -> arg
   -> Command arg res
-mkCommand dict actionPtr arg =
+mkCommand dict action arg =
```

Comment:
Unrelated change?

Reply:
The commit with this change is "Avoid -Wname-shadowing".

Reply:
I've added this to the description. I dropped the `Ptr` suffix from one rename and looked up the `Note [Hooks Binary instances]` to pick the other name, `dynDeps`. Thanks for leaving the note, @sheaf.

https://github.com/haskell/cabal/blob/949464d333b573efb47bd84c679d4c86b21526c9/Cabal/src/Distribution/Simple/SetupHooks/Rule.hs#L729

Reply:
Assume, thank you!